e3sm_diags/plot/cartopy/polar_plot.py

```diff
@@ -1,278 +0,0 @@
-from __future__ import print_function
-
-import os
-
-import cartopy.crs as ccrs
-import matplotlib
-import matplotlib.pyplot as plt
-import numpy as np
-import numpy.ma as ma
-
-from e3sm_diags.driver.utils.general import get_output_dir
-from e3sm_diags.logger import custom_logger
-from e3sm_diags.plot import get_colormap
-
-matplotlib.use("Agg")
-import matplotlib.colors as colors  # isort:skip  # noqa: E402
-import matplotlib.path as mpath  # isort:skip  # noqa: E402
-
-logger = custom_logger(__name__)
-
-plotTitle = {"fontsize": 11.5}
-plotSideTitle = {"fontsize": 9.0}
-
-# Position and sizes of subplot axes in page coordinates (0 to 1)
-panel = [
-    (0.27, 0.65, 0.3235, 0.25),
-    (0.27, 0.35, 0.3235, 0.25),
-    (0.27, 0.05, 0.3235, 0.25),
-]
-
-# Border padding relative to subplot axes for saving individual panels
-# (left, bottom, right, top) in page coordinates
-border = (-0.02, -0.01, 0.14, 0.04)
-
-
-def add_cyclic(var):
-    lon = var.getLongitude()
-    return var(longitude=(lon[0], lon[0] + 360.0, "coe"))
-
-
-def get_ax_size(fig, ax):
-    bbox = ax.get_window_extent().transformed(fig.dpi_scale_trans.inverted())
-    width, height = bbox.width, bbox.height
-    width *= fig.dpi
-    height *= fig.dpi
-    return width, height
```

Comment:
Doesn't appear to have a correspondence. But  the results of its one call also doesn't appear to be used.

Reply:
I noticed this was duplicated in various plotter files, but they were never referenced. VS Code makes it easy to check for references.